core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2651,6 +2651,16 @@ public String toString() {
       .setConsistencyCheckLevel(ConsistencyCheckLevel.ENFORCE)
       .setScope(Scope.WORKER)
       .build();
+  public static final PropertyKey WORKER_MASTER_PERIODICAL_CONNECT_TIMEOUT =
```

Comment:
It can be somehow confusing to have both `WORKER_MASTER_PERIODICAL_CONNECT_TIMEOUT` and `WORKER_MASTER_CONNECT_RETRY_TIMEOUT` in the configuration. Can we have a more distinguishable name for this one? How about `WORKER_MASTER_PERIODICAL_RPC_TIMEOUT`?

Reply:
Sounds good